frontend/packages/dev-console/console-extensions.json

```diff
@@ -186,6 +186,21 @@
       ]
     }
   },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "project-access",
+      "href": "/project-access/ns/:namespace",
+      "label": "%devconsole~Sharing%",
+      "description": "%devconsole~Project access allows you to add or remove a user's access to the project.%",
+      "icon": { "$codeRef": "icons.shareProjectIconElement" },
+      "accessReview": [
+        { "group": "rbac.authorization.k8s.io", "resource": "rolebindings", "verb": "create" },
+        { "group": "rbac.authorization.k8s.io", "resource": "rolebindings", "verb": "list" }
+      ],
+      "insertBefore": "local-machine"
```

Comment:
I don't see `insertBefore` in the definition for type `dev-console.add/action` here https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/extensions/add-actions.ts#L5-L23